src/components/BaseCard.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="card">
+    <div v-if="title" class="title">
+      <h5 class="card-title">{{ title }}</h5>
+    </div>
+    <slot></slot>
+  </div>
+</template>
+<script>
+export default {
+  name: 'baseCard',
+  props: {
+    title: String,
+  },
+}
+</script>
+<style lang="scss">
+  @import '@/styles/_colors.scss';
+  @import '@/styles/theme.scss';
+  .card {
+    max-width: 600px;
```

Comment:
Why do you _max-width_ instead of width? We want to resize the card component or we prefer add a fixed size?